external-crates/move/crates/move-compiler/tests/sui_mode/linter/no_trigger_lint_missing_key.move

```diff
@@ -0,0 +1,43 @@
+module a::no_trigger_lint_cases {
+    use sui::object::UID;
+
+    // This should not trigger the linter warning (true negative)
+    struct HasKeyAbility has key {
+        id: UID,
+    }
+
+    // False positive cases (should not trigger warning but might):
```

Comment:
These are not really false positives. None of these cases trigger the warning erroneously--please remove